src/coreclr/nativeaot/BuildIntegration/DotNetJsApi.targets

```diff
@@ -0,0 +1,146 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
This is clearly a copy from somewhere else. Is there a plan for un-copying it?

If not (which is reasonable, it's not easy to do), what is the plan to ensure it stays up-to-date with merges? How often do these compilation options change? How easy is to detect breaks from something missing?

If there is, it needs to be figured out how things that I would expect not to apply to NAOT can be segregated. There are a lot of them.

Reply:
I'm happy with the copy at the moment. I don't think these parameters change often. 

Original source of these values are `wasm.proj` and `WasmApp.Native.targets`. With mono we run emscripten with default configuration during runtime build and ship the outputs for usage without AOT (in this mode user doesn't need emscripten on his machine). We also ship the rsp files and json with configuration values. When "native build" (emscripten) is run during target app build, these default values are mixed with user defined parameters (applied in WasmApp.Native.targets).

I initially tried to mimic the same behavior (for parameter flow), but failed due to various reasons. I plan to come back to it and find a better way as we progress.

Reply:
Alright, if these don't change too often keeping a copy looks good to me. I will comment on the individual items then.